src/main.rs

```diff
@@ -12,7 +12,7 @@ use trin_core::{
     cli::{TrinConfig, HISTORY_NETWORK, STATE_NETWORK},
     jsonrpc::service::launch_jsonrpc_server,
     portalnet::{discovery::Discovery, types::PortalnetConfig},
-    utils::setup_overlay_db,
+    utils::utils::setup_overlay_db,
```

Comment:
not for this PR, but I don't love `utils::utils`. Once you merge this I'm going to make a `utils/db.rs` that will include this function, and then we can think of a better name for a file containing the other things.